Src/java/elm-fhir/src/main/java/org/cqframework/cql/elm/requirements/fhir/utilities/constants/CrmiConstants.java

```diff
@@ -0,0 +1,17 @@
+package org.cqframework.cql.elm.requirements.fhir.utilities.constants;
+
+public class CrmiConstants {
+
+    private CrmiConstants() {}
+
+    public static final String EFFECTIVE_DATA_REQUIREMENTS_EXT_URL =
+            "http://hl7.org/fhir/uv/crmi/StructureDefinition/crmi-effectiveDataRequirements";
+    public static final String EFFECTIVE_DATA_REQUIREMENTS_IDENTIFIER = "effective-data-requirements";
+
+    public static final String SOFTWARE_SYSTEM_EXT_URL =
+            "http://hl7.org/fhir/uv/crmi/StructureDefinition/crmi-softwaresystem";
+    public static final String SOFTWARE_SYSTEM_DEVICE_PROFILE_URL =
+            "http://hl7.org/fhir/uv/crmi/StructureDefinition/crmi-softwaresystemdevice";
+    public static final String SOFTWARE_SYSTEM_DEVICE_TYPE_SYSTEM_URL =
+            "http://terminology.hl7.org/ValueSet/software-system-type";
```

Comment:
This should be `http://terminology.hl7.org/CodeSystem/software-system-type-codes`, right?

Reply:
Yep, fixed, thank you